packages/extension/src/components/organisms/NftListing/NftListing.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+
+import { List } from '@mui/material';
+import InfiniteScroll from 'react-infinite-scroll-component';
+
+import { AccountNFTokenResponse } from '@gemwallet/constants';
+
+import { InformationMessage } from '../../molecules';
+import { NftCard } from '../../molecules/NftCard';
+
+export interface NftListingProps extends AccountNFTokenResponse {
+  onLoadMoreClick: () => void;
+}
+
+export const NftListing: FC<NftListingProps> = ({ account_nfts, marker, onLoadMoreClick }) => {
+  if (account_nfts.length === 0) {
```

Comment:
You need a loading state, when I am going to the NFTListing page, I can see the InformationMessage and then a list of NFTs.

Reply:
added